src/phases/execution.rs

```diff
@@ -45,3 +46,66 @@ pub fn run_tests(rx: mpsc::Receiver<TestCase>, tx: mpsc::Sender<TestCase>) -> Re
 
     Ok(())
 }
+
+pub fn get_parametrizations(path: &str, name: &str) -> Result<Vec<String>> {
```

Comment:
I put this under `execution.rs` even though this is a collection step as it kind of made sense to keep code bound logic to be in a single place but definitely open to suggestions

Reply:
I think that makes sense.